frontend/components/site/SiteScans.jsx

```diff
@@ -42,14 +44,13 @@ function SiteScans() {
     return thisMonthScan;
   }
 
-  const filteredScans = (scansList = scans) => scansList
-  // fake adding artifacts, delete next line before merge
-    .map(scan => (Number.isInteger(scan.count) ? { ...scan, artifact: { url: '#foo', size: 12345 } } : scan))
-    .filter(scan => scan.status !== 'cancelled');
-
-  if (!site || !scans) {
-    return null;
-  }
+  if (!site || !scans) return null;
+  const branchToBeScanned = site?.defaultBranch || 'most recently built';
+  const buildIdToFilterBy = Math.abs(parseInt(searchParams.get('build'), 10));
```

Comment:
since it's a parameter controlled by us (via the router), you can safely do `Number(searchParams.get('build'))`

Reply:
people can type `'build=-1` or `build=foo` in the URL bar